actions.js

```diff
@@ -1,876 +1,947 @@
+			name: 'Jump to selected cue',
+			options: [],
+			callback: () => {
+				this.sendCommand('jumpToSelectedCue')
+			},
+		},
+		select_prev: {
+			name: 'Select previous cue',
+			options: [],
+			callback: () => {
+				this.sendCommand('selectPrevCue')
+			},
+		},
+		select_next: {
+			name: 'Select next cue',
+			options: [],
+			callback: () => {
+				this.sendCommand('selectNextCue')
+			},
+		},
+		goto_30: {
+			name: 'Goto 30',
+			options: [],
+			callback: () => {
+				this.sendCommand('goto30')
+			},
+		},
+		goto_20: {
+			name: 'Goto 20',
+			options: [],
+			callback: () => {
+				this.sendCommand('goto20')
+			},
+		},
+		goto_10: {
+			name: 'Goto 10',
+			options: [],
+			callback: () => {
+				this.sendCommand('goto10')
+			},
+		},
+		play_select: {
+			name: 'Play Selected Cue',
+			options: [],
+			callback: () => {
+				this.sendCommand('playSelectedCue')
+			},
+		},
+		fullscreenToggle: {
+			name: 'Toggle Fullscreen',
+			options: [],
+			callback: () => {
+				this.sendCommand('toggleFullscreen')
+			},
+		},
+		fullscreenOn: {
+			name: 'Fullscreen On',
+			options: [],
+			callback: () => {
+				this.sendCommand('fullscreenOn')
+			},
+		},
+		fullscreenOff: {
+			name: 'Fullscreen Off',
+			options: [],
+			callback: () => {
+				this.sendCommand('fullscreenOff')
+			},
+		},
+		plLoopToggle: {
+			name: 'Toggle Playlist Loop',
+			options: [],
+			callback: () => {
+				this.sendCommand('toggleLoop')
+			},
+		},
+		plLoopOn: {
+			name: 'Playlist Loop On',
+			options: [],
+			callback: () => {
+				this.sendCommand('loopOn')
+			},
+		},
+		plLoopOff: {
+			name: 'Playlist Loop Off',
+			options: [],
+			callback: () => {
+				this.sendCommand('loopOff')
+			},
+		},
+		plTransToggle: {
+			name: 'Toggle Playlist Transition on Play',
+			options: [],
+			callback: () => {
+				this.sendCommand('toggleTransitionOnPlay')
+			},
+		},
+		plTransOff: {
+			name: 'Transition on Play Off',
+			options: [],
+			callback: () => {
+				this.sendCommand('transitionOnPlayOff')
+			},
+		},
+		plTransOn: {
+			name: 'Transition on Play On',
+			options: [],
+			callback: () => {
+				this.sendCommand('transitionOnPlayOn')
+			},
+		},
+		resendOSCFeedback: {
+			name: 'Resend OSC feedback',
+			options: [],
+			callback: () => {
+				this.sendCommand('resendOSCFeedback')
+			},
+		},
+		jump_cue: {
+			name: 'Jump to specific cue',
+			options: [
+				{
+					type: 'textinput',
+					useVariables: true,
+					label: 'Cue number or ID',
+					id: 'cuenumber',
+					default: 'current',
+				},
+			],
+			callback: async (action) => {
+				this.sendCommand(`${await this.conformCueID(action.options.cuenumber)}/jump`)
+			},
+		},
+		jumpCueName: {
+			name: 'Jump to cue with name',
+			options: [
+				{
+					type: 'textinput',
+					useVariables: true,
+					label: 'Cue Name',
+					id: 'string',
+				},
+			],
+			callback: (action) => {
+				this.parseVariablesInString(action.options.string).then((value) => {
```

Comment:
You could write this as:
```
callback: async (action) => {
	const value = await this.parseVariablesInString(action.options.string)
	this.sendCommand('jumpToCueWithName', value)
},
```

Reply:
@Julusian Addressed this with https://github.com/bitfocus/companion-module-imimot-mitti/pull/25